libraries/dagster-unity-catalog-polars/dagster_unity_catalog_polars/unity_catalog_iomanager.py

```diff
@@ -0,0 +1,118 @@
+from collections.abc import Callable
+from functools import wraps
+
+import polars as pl
+from dagster import (
+    ConfigurableIOManager,
+    InputContext,
+)
+from pydantic import ConfigDict
+
+from .utils import read_uc_table
+
+
+class UnityCatalogInputManager(ConfigurableIOManager):
```

Comment:
```suggestion
class DatabricksUnityCatalogInputManager(ConfigurableIOManager):
```
Since it's only for databricks ecosystem.

Reply:
done